controlpad/controller/controlpad.js

```diff
@@ -30,9 +29,12 @@ ws.onopen = (_event) => {
             }
         }
         else {
-            let msg = event.data;
-            console.log("received message" + '<' + msg + '>');
-            console.log('^^ WASNT EXPECTING TO RECV ANY MSGS ^^');
+            const data = JSON.stringify(event.data);
```

Comment:
Ideally this file shouldn't me modified. 

This file will eventually become just a symbolic link on the console and any custom changes here won't be reflected. 

I can export data and create a handle message in the app.js if that's better.

Reply:
Yes I need some way to ge the information out. Right, at 
least on this code I don't have access to message, as far as I can tell